docker-compose.yml

```diff
@@ -36,3 +36,8 @@ services:
          - .docker/phpmyadmin/conf.inc.php:/etc/phpmyadmin/conf.inc.php
         ports:
         - "8080:80"
+
+    composer:
+        image: composer
+        volumes:
+         - ./:/app
```

Comment:
Did you try to use this container ?
How do you use it ?

Reply:
Oui pour la première question et cela permet d'installer les vendors. 
`docker-compose run composer` pour l'utiliser